### PR TITLE
Add missing steps

### PR DESCRIPTION
**Pull request by sfiligoi**
The original description would not result in a working system.
With the proposed changes, the setup works.

### CODE REVIEW

**Comment by sfiligoi** [comment]
I signed the CLA... not sure why it still claims I did not.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-19482--kubernetes-io-master-staging.netlify.com/docs/tasks/administer-cluster/configure-multiple-schedulers/

/cc @kubernetes/sig-scheduling-pr-reviews

**Comment by sftim** [comment]
@sfiligoi this will need technical review before it merges. From a Markdown point-of-view it's good enough to merge.

**Comment by kbhawkey** [comment]
Hi @Huang-Wei , Would you have time to provide a technical review or could you suggest another member of @kubernetes/sig-scheduling-pr-reviews ? Thank you.

File: content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md [hunk 1]
```diff
@@ -84,6 +84,11 @@ Please see the
 [kube-scheduler documentation](/docs/admin/kube-scheduler/) for
 detailed description of other command line arguments.
 
+You will also need to manually create the appropriate rolebinding:
+```
+kubectl create rolebinding -n kube-system ROLEBINDING_NAME --role=extension-apiserver-authentication-reader --serviceaccount=kube-system:my-scheduler
+```
+
```
Comment (Huang-Wei) [inline content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: I suppose the `extension-apiserver-authentication-reader` role is a kubeadm thing? I apologize I'm not quite familiar with it. @neolit123, could you kindly confirm?

(if it's specific with kubeadm, I think we shouldn't include the step here)

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: hi, no `extension-apiserver-authentication-reader` is not a role that kubeadm defines. i can see it being mentioned in this guide:
https://kubernetes.io/docs/tasks/access-kubernetes-api/setup-extension-api-server/

@drnic 
i don't think setting up multiple schedulers requires the api-server extensions mechanism.
could you explain why this role is used here?

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: if this role is truly required, please add a comment clarifying this + providing external link.

Comment (sfiligoi) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: I am not a Kubernetes expert.
The original instructions would not work on a bare-metal setup.
But the update ones do. That's really all I know.

Comment (zacharysarah) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: @sfiligoi Thanks for this PR. This PR can only advance with a more thorough technical explanation, which you must provide. Please advise on how you'd like to proceed, or whether you'd like to close this PR.

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: @sfiligoi
> But the updated ones do. That's really all I know.

could you provide the source from where you got this information?

the `extension-apiserver-authentication-reader` role grants read access to the `extension-apiserver-authentication` config map in kube-system. [this is by design](https://kubernetes.io/docs/tasks/access-kubernetes-api/#kubernetes-apiserver-client-authentication) for ensuring the extended api-server can validation requests, yet it's still not clear to me why we need to grant the same access to the second scheduler service account.

by binding the service account to `system:kube-scheduler`, the second scheduler should have sufficient privileges. this was added here https://github.com/kubernetes/website/commit/d03745376c281a37a3dd3cceeb1c2fb86ad415cb

@kubernetes/sig-api-machinery-pr-reviews

Comment (sfiligoi) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: @neolit123 
Will dig out where I got the information;
it was the sum of a couple of forums. (was a month ago... should have really added that info in the original PR!)

But I confirm the original instruction did not work on the day I submitted the PR.
Can someone else try to follow the original instructions on a bare-metal node (CentOS), and either confirm or contradict me?

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md:91]: @sfiligoi this sounds to me like it's worth investigating and filing an issue against the documentation

I'm not convinced that there is a good case (yet) for making the changes you propose. Would you be willing to file that issue and to continue the discussion there? Once it's clear what the problem is and roughly how to fix it, that's when I think it makes sense to open a pull request.

**Comment by jimangel** [comment]
@sfiligoi Thanks for your PR! I'm going to close this PR until there's additional technical justification. Feel free to reopen if you feel that I'm closing it in error.

/close

**Pull request closed without merging**